dask/dataframe/tests/test_io.py

```diff
@@ -668,3 +668,29 @@ def test_report_dtype_correction_on_csvs():
             assert False
         except ValueError as e:
             assert "'numbers': 'float64'" in str(e)
+
+
+def test_hdf_globbing():
+    pytest.importorskip('tables')
+    df = pd.DataFrame({'x': ['a', 'b', 'c', 'd'],
+                       'y': [1, 2, 3, 4]}, index=[1., 2., 3., 4.])
+    a = dd.from_pandas(df, 2)
+
+    pattern = '/tmp/*foo.h5'
```

Comment:
I stored it as a variable because I wanted to make it os independent later. However I'm not really sure how `tempfile` works on windows so I'll skip this until someone complains.

I'll inline it and change the name to be more unique.